src/decoders/mod.rs

```diff
@@ -186,12 +186,43 @@ impl Camera {
   }
 }
 
-pub fn ok_image(camera: &Camera, width: usize, height: usize, wb_coeffs: [f32;4], image: Vec<u16>) -> Result<RawImage,String> {
-  Ok(RawImage::new(camera, width, height, wb_coeffs, image))
+/// Possible values for the IFD tag Orientation (0x0112)
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum Orientation {
```

Comment:
I just tried some of the uncommon ones in Adobe CameraRaw and macOS preview. Both of them respected `IFD0:Orientation` for both CR2 and TIFF. I tried both vertical and horisontal flipping. As for a camera actually producing images with these values I kind of doubt it.

I think it makes little sense to not allow it, as the functionality is already there and it doesn't make the handling any more complicated.